protected/models/Patient.php

```diff
@@ -190,7 +190,7 @@ public function attributeLabels()
         return array(
             'id' => 'ID',
             'pas_key' => 'PAS Key',
-            'dob' => 'Date of Birth',
+            'dob' => 'Date of Birth <i style="color:grey">(01 Jan 1970)</i>',
```

Comment:
Alternatively, you could add a placeholder html property to the text field itself.